wolfcrypt/src/tfm.c

```diff
@@ -5483,6 +5483,13 @@ int mp_radix_size (mp_int *a, int radix, int *size)
     }
     fp_zero (t);
 
+#ifndef WC_DISABLE_RADIX_ZERO_PAD
+    /* For hexadecimal output, add zero padding when number of digits is odd */
+    if ((digs & 1) && (radix == 16)) {
```

Comment:
This won't work when the number is negative.

Reply:
Good catch, thanks Sean.